config/firestore.rules

```diff
@@ -0,0 +1,18 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /datasets/covid19-som/metadata/{document=**} {
+    	allow write: if
+      	request.auth.token.email_verified &&
```

Comment:
probably not strictly necessary, but consider adding a null check to check specifically for unauthenticated access:
```suggestion
        request.auth != null &&
      	request.auth.token.email_verified &&
```

Reply:
Ah sure, will do